india_banking_connector/connectors/doctype/indusind_bank_connector/indusind_bank_connector.py

```
@@ -0,0 +1,514 @@
+			"Content-Type": "application/json",
+		}
+
+	# ---- operations (poll-based: TransactionPosting ack, then TransactionEnquiry) ----
+
+	def initiate_payment(self):
+		unique_id = self._unique_id()
+		if existing := self.validate_duplicate_payments(unique_id=unique_id):
+			return existing
+
+		payload = self.get_encrypted_payload("make_payment", lower=True)
+		response = requests.post(self.urls.make_payment, headers=self.headers, json=payload)
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)